### PR TITLE
Delete Changes

### PR DESCRIPTION
**Pull request by zpnk**
504abb5 is a refactor to clean up the ChangePage component.
cbfea25 adds the deletion functionality.
c045d15 adds the sequelize-cli migration for the required `deletedAt` field.

`./node_modules/.bin/sequelize db:migrate` runs the migration. May need to edit `config/config.json`.

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "development": {
+    "username": "changepane",
```
Comment (amccloud) [inline config/config.json]: Sequelize supports pulling the database connection information from the environment.

```
"use_env_variable": "DATABASE_URL"
```

Comment (zpnk) [inline config/config.json]: Ah yeah, I was digger more into the docs on this yesterday. It also supports a `.sequelizerc` file that may be useful. I'm going to play around with it some more.

**Comment by zpnk** [comment]
Ready for re-review! See f073bf9 for the new migration config.

If this checks out, feel free to merge.

**Comment by zpnk** [comment]
Fixed!

**Pull request merged**